### PR TITLE
[VEN-2495]: VIP for Oracle Configuration on Arbitrum Sepolia

### PR DESCRIPTION
**Pull request by Debugger022**
## Description

Resolves #VEN-2495
This **VIP** configures the oracle on **Arbitrum Sepolia** network.
Adds the feeds for following tokens:
- MOCK-USDC
- MOCK_USDT
- MOCK_WETH
- MOCK_WBTC
- MOCK_ARB

and sets direct price of 10$ for **XVS**.

### CODE REVIEW

File: src/networkAddresses.ts [hunk 1]
```diff
@@ -111,6 +111,27 @@ export const NETWORK_ADDRESSES = {
...
+    MOCK_USDC: "0xDb3EC247767d586a9B77F7472F3C7759b131bE69",
+    MOCK_USDT: "0xE1F20e585B57F82d57Da92ac1639B634f56D77CF",
+    MOCK_WETH: "0x980B62Da83eFf3D4576C647993b0c1D7faf17c73",
+    MOCK_WBTC: "0xD47e7683a11d70E479bb948dc8E795778035C62D",
+    MOCK_ARB: "0x4371bb358aB5cC192E481543417D2F67b8781731",
+    XVS: "0x47fA6E9F717c9eB081c4734FfB5a1EcD70508891",
+    POOL_REGISTRY: "0xEcFa4d4FcBB76A19EB90F947CCCd5c29375bD3Bd",
+    COMPTROLLER_CORE: "0x5e5221f13b50Bc93f0DdD995911360807f48892e",
+    VUSDC_CORE: "0x4511CFAB7CF70BC6cbEEddb8972E73806813b3bf",
+    VUSDT_CORE: "0xA4D749a3eEfA87c03eDc7CD107676452d5712979",
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply src/networkAddresses.ts]: https://github.com/VenusProtocol/vips/pull/247/commits/0a710c85af6815d4394d915886a5c57d22bddf6f

File: multisig/proposals/arbitrumsepolia/vip-001/index.ts [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+    },
+    {
+      target: BOUND_VALIDATOR,
+      signature: "acceptOwnership()",
+      params: [],
+    },
+    {
+      target: arbitrumsepolia.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[MOCK_WBTC, CHAINLINK_BTC_FEED, 144000]],
```
Comment (chechu) [inline multisig/proposals/arbitrumsepolia/vip-001/index.ts:81]: The heartbeat for BTC/USD in Arbitrum sepolia is `120`. `144` seconds seems enough, wdyt?

Same comment for ETH/USD

File: multisig/proposals/arbitrumsepolia/vip-001/index.ts [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
+    {
+      target: arbitrumsepolia.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[MOCK_USDC, CHAINLINK_USDC_FEED, 144000]],
```
Comment (chechu) [inline multisig/proposals/arbitrumsepolia/vip-001/index.ts:121]: We are usually setting a stale period of 26h (93600 seconds) for feeds with a heartbeat of 24h, like this one

File: multisig/proposals/arbitrumsepolia/vip-001/index.ts [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
+    {
+      target: arbitrumsepolia.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[MOCK_ARB, CHAINLINK_ARB_FEED, 144000]],
```
Comment (chechu) [inline multisig/proposals/arbitrumsepolia/vip-001/index.ts:161]: I would set 4200 seconds for the stale period here (heartbeat of 1 hour)

File: multisig/proposals/arbitrumsepolia/vip-001/index.ts (hunk 2 above)
Comment (Debugger022) [reply multisig/proposals/arbitrumsepolia/vip-001/index.ts:82]: I think setting `144` seconds would be enough. But as i can see that for 1 minute heartbeat feeds, we have set it at 100 seconds [here](https://github.com/VenusProtocol/vips/blob/b8325d5d0a61136f12cac100b4c8540f1d1b6c93/multisig/proposals/opbnbmainnet/vip-001/index.ts#L16)
So, what if we set it at `180` seconds?

File: multisig/proposals/arbitrumsepolia/vip-001/index.ts (hunk 4 above)
Comment (Debugger022) [reply multisig/proposals/arbitrumsepolia/vip-001/index.ts:162]: What if we set it at `6000`seconds(`100`minutes) as we have set [here](https://github.com/VenusProtocol/vips/blob/b8325d5d0a61136f12cac100b4c8540f1d1b6c93/multisig/proposals/ethereum/vip-001/index.ts#L10)?

Comment (Debugger022) [reply multisig/proposals/arbitrumsepolia/vip-001/index.ts]: Done https://github.com/VenusProtocol/vips/pull/247/commits/c573c14b8980fedd4ab11d46d064198f238595b3

File: multisig/proposals/arbitrumsepolia/vip-001/index.ts (hunk 3 above)
Comment (Debugger022) [reply multisig/proposals/arbitrumsepolia/vip-001/index.ts]: Done https://github.com/VenusProtocol/vips/pull/247/commits/c573c14b8980fedd4ab11d46d064198f238595b3

File: multisig/proposals/arbitrumsepolia/vip-001/index.ts (hunk 2 above)
Comment (Debugger022) [reply multisig/proposals/arbitrumsepolia/vip-001/index.ts]: Done https://github.com/VenusProtocol/vips/pull/247/commits/c573c14b8980fedd4ab11d46d064198f238595b3

**Review by chechu — APPROVED** [review]

**Comment by Debugger022** [comment]
Multisig tx: https://sepolia.arbiscan.io/tx/0xebfb689dc71cdde75f92a2767983ba43756697cb07d5def74d9868c70b4afcd4

**Pull request merged**